config/urls.py

```diff
@@ -5,5 +5,5 @@
 
 urlpatterns = [
     path('photo', include('photo.urls')),
-    path('account', include('account.urls'))
+    path('', include('account.urls'))
```

Comment:
```suggestion
    path('account', include('account.urls'))
```
바로 account.urls로 연결하는 것보다는 url에 앱 이름을 명시해주는 것이 좋습니다.

Reply:
넹